packages/cli/src/create.ts

```diff
@@ -127,16 +127,23 @@ export async function create(): Promise<void> {
 async function promptForResourceCreation(ccfg: NewRenderedClusterConfigType) {
   if (ccfg.cloud_provider === "aws") {
     const url = `https://go.opstrace.com/cli-aws-mutating-api-calls/${BUILD_INFO.VERSION_STRING}`;
+    const tos = "https://go.opstrace.com/terms-of-service";
+    // TODO omit AWS API link if a dev build
     log.info(
-      "Before we continue, please review the set of state-mutating " +
-        "AWS API calls emitted by this CLI during cluster creation: %s",
-      url
+      "\n\nYou can review the set of state-mutating AWS API calls made by this CLI: %s\n\n" +
+        "By selecting 'y' you agree to our terms of service: %s\n\n",
+      url,
+      tos
     );
-    await util.promptForProceed();
   }
 
-  // if (ccfg.cloud_provider === "gcp") {
-  // }
+  if (ccfg.cloud_provider === "gcp") {
```

Comment:
I added this here in order to have all messages related to the prompt in a single message to make it as easy-to-read as possible.